public/js/vulx.profile.js

```diff
@@ -160,12 +160,21 @@ for(var i = 0; i < ranksJson.length; i++) {
     rankImg.setAttribute("style", "height: 30px;");
     rankImg.setAttribute("class", "valorantRankImg");
     rankImg.setAttribute("src", `https://cdn.aquaplays.xyz/ranks/${ranksJson[i].id < 3 ? 0 : ranksJson[i].id}.png`);
-    rank.appendChild(rankImg);
+    //rank.appendChild(rankImg);
```

Comment:
unnecessary code comment, can be safely deleted

Reply:
Applied your changes ♥